ergo-wallet/src/main/scala/org/ergoplatform/wallet/transactions/TransactionBuilder.scala

```diff
@@ -68,7 +68,7 @@ object TransactionBuilder {
     dataInputs: IndexedSeq[DataInput],
     outputCandidates: Seq[ErgoBoxCandidate],
     currentHeight: Int,
-    feeAmount: Long,
+    createFeeOutput: Option[Long],
```

Comment:
feeAmountOpt: Option[Long]

"create..." sounds like a function or a boolean parameter  name

Reply:
yeah, but feeAmountOpt hides the semantics of why it is optional, i.e. that when it is None, then new feeOut is not created, and then it is Some, then it is created. That is why "create" in the name.
I suggest to leave it with "create", because parameter is hard to confuse with method.